frontend/src/components/common/courses-list/components/course/course.tsx

```diff
@@ -12,16 +12,17 @@ import styles from './styles.module.scss';
 
 type Props = {
   course: CourseGetResponseDto;
+  isPopular?: boolean;
 };
 
-const Course: FC<Props> = ({ course }) => {
+const Course: FC<Props> = ({ course, isPopular = false }) => {
   const keyNameKebabCase = changeStringCase({
     stringToChange: course.category?.key as string,
     caseType: StringCase.KEBAB_CASE,
   });
 
   return (
-    <li className={styles.container}>
+    <div className={styles.container}>
```

Comment:
@Liza-Veis there's an issue with `li`. Because `react-multi-carousel` rendes each element as `li`, we'll get an error `validateDOMNesting`, smth like `cannot render li as a descendant of li`
As far as I know, it's totally okay to render `div` inside of `ul`

Reply:
got it